gpt_index/indices/composability/graph.py

```diff
@@ -194,10 +252,18 @@ def load_from_string(cls, index_string: str, **kwargs: Any) -> "ComposableGraph"
         from gpt_index.indices.registry import load_index_struct_from_dict
 
         result_dict = json.loads(index_string)
-        index_struct = load_index_struct_from_dict(result_dict["index_struct"])
-        docstore = DocumentStore.load_from_dict(result_dict["docstore"])
+        index_struct = load_index_struct_from_dict(result_dict[INDEX_STRUCT_KEY])
+        docstore = DocumentStore.load_from_dict(result_dict[DOCSTORE_KEY])
+        additional_query_context = load_additional_query_context_from_dict(
```

Comment:
so right now, load_from_dict would break for any vector store where `from_dict` is not implemented yet right?

Reply:
whereas before at least you could construct the vector store manually and pass it into query_kwargs. so rn this would mean that composing on top of pinecone indices, and then save/load from disk, would be broken

Reply:
Agreed with concerns here. A better outcome is the original behavior, where even if the vector store cannot be properly loaded from the save dict, we should still allow user to pass the index back in after the fact.

Reply:
RE second point, yes currently it will break if we passed in pinecone_index when constructing the Index.

Reply:
This is now fixed.